Add `New Featured Artist: Marmalade butcher` news post (#6449)
ppy/osu-wiki: news/2021-11-27-new-featured-artist-marmalade-butcher.md

---
layout: post
title: "New Featured Artist: Marmalade butcher"
date: 2021-11-27 09:00:00 +0000
---

Our newest Featured Artist **Marmalade butcher** hits you in the face with their innovative math rock sound!

> ![](https://assets.ppy.sh/artists/219/header.jpg)

Reviewer: 220 not 219

**Marmalade butcher** may have only received their first ranked map a few months ago, but their presence in osu! has been ever-present. Take a peek at the intensity of their tracks through a video of [*Marmalade butcher - Y/M/K/M*](https://osu.ppy.sh/beatmapsets/1502963), then try out any of the other awesome beatmaps further below:

<div align="center">
    <video width="100%" controls>
        <source src="https://assets.ppy.sh/artists/220/release_showcase.mp4" type="video/mp4" preload="none">
    </video>
</div>

### Marmalade butcher - Y/M/K/M

Experience [the beatmap from the video above](https://osu.ppy.sh/beatmapsets/1502963) hosted by [Lasse](https://osu.ppy.sh/users/896613)!

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

### Marmalade butcher - Ushinahareru Runazuki

See how **Marmalade butcher** translates to osu!mania through [this beatmap](https://osu.ppy.sh/beatmapsets/1486395) hosted by [Shoegazer](https://osu.ppy.sh/users/2520707)!

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

### Marmalade butcher - Amanita

> Check out [this popular graveyard map](https://osu.ppy.sh/beatmapsets/1139182) hosted by [xbopost](https://osu.ppy.sh/users/6842421) or [this map from OWC 2020](https://osu.ppy.sh/beatmapsets/1306572) created by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mao](https://osu.ppy.sh/users/2204515), and [Zelq](https://osu.ppy.sh/users/8953955)!

Reviewer: iirc Panda's Amanita have been moved to http://osu.ppy.sh/beatmapsets/1473778#osu/3079004 so i think it would be more accurate to phrase this as something like this :

`...or [this map](https://osu.ppy.sh/beatmapsets/1473778) hosted by [Camo](https://osu.ppy.sh/users/5194391) which contains the distinguished OWC 2020 difficulty made by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mao](https://osu.ppy.sh/users/2204515), and [Zelq](https://osu.ppy.sh/users/8953955)!`

Author: did it, but made it a bit easier to read

---
layout: post
title: "New Featured Artist: Marmalade butcher"
date: 2021-11-27 09:00:00 +0000
---

Our newest Featured Artist **Marmalade butcher** hits you in the face with their innovative math rock sound!

![](https://assets.ppy.sh/artists/220/header.jpg)

> Blending some of music's less conventional genres into their own soup of aural magic, **Marmalade butcher** is a next-generation hybrid band that took flight in 2010. Renowned for their complex sound that drifts from the usual electronic fare from rhythm games, this group is the ideal change for anyone and everyone in the osu! mapping scene.

Reviewer 2: clearer wording (fare?)

```suggestion
Blending some of music's less conventional genres into their own soup of aural magic, **Marmalade butcher** is a next-generation hybrid band that took flight in 2010. Renowned for their complex sound that drifts away from the usual electronic vibe of rhythm games, this group is the ideal change for anyone and everyone in the osu! mapping scene.
```

**Marmalade butcher** may have only received their first ranked map a few months ago, but their presence in osu! has been ever-present. Take a peek at the intensity of their tracks through a video of [*Marmalade butcher - Y/M/K/M*](https://osu.ppy.sh/beatmapsets/1502963), then try out any of the other awesome beatmaps further below:

<div align="center">
    <video width="100%" controls>
        <source src="https://assets.ppy.sh/artists/220/release_showcase.mp4" type="video/mp4" preload="none">
    </video>
</div>

### Marmalade butcher - Y/M/K/M

Experience [the beatmap from the video above](https://osu.ppy.sh/beatmapsets/1502963) hosted by [Lasse](https://osu.ppy.sh/users/896613)!

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

### Marmalade butcher - Ushinahareru Runazuki

See how **Marmalade butcher** translates to osu!mania through [this beatmap](https://osu.ppy.sh/beatmapsets/1486395) hosted by [Shoegazer](https://osu.ppy.sh/users/2520707)!

<audio controls>
    <source src="LINK" type="audio/mpeg">
</audio>

### Marmalade butcher - Amanita

Check out one of these masterpieces:

> - [popular graveyard map](https://osu.ppy.sh/beatmapsets/1139182) hosted by [xbopost](https://osu.ppy.sh/users/6842421)

Reviewer 2: ```suggestion
- [Popular graveyard map](https://osu.ppy.sh/beatmapsets/1139182) hosted by [xbopost](https://osu.ppy.sh/users/6842421)
```

> - [full mapset spread](https://osu.ppy.sh/beatmapsets/1473778) hosted by [Camo](https://osu.ppy.sh/users/5194391) and including an awesome OWC 2020 difficulty made by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mao](https://osu.ppy.sh/users/2204515), and [Zelq](https://osu.ppy.sh/users/8953955)!

Reviewer 2: ```suggestion
- [full mapset spread](https://osu.ppy.sh/beatmapsets/1473778) hosted by [Camo](https://osu.ppy.sh/users/5194391), including an awesome OWC 2020 difficulty made by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mao](https://osu.ppy.sh/users/2204515), and [Zelq](https://osu.ppy.sh/users/8953955)!
```
